src/types/models/dealerPromotion.ts

```diff
@@ -0,0 +1,13 @@
+interface Data {
+  description?: string
+  title: string
+}
+
+export interface DealerPromotion {
+  dataDe?: Data
```

Comment:
Is this naming enforced by BE? Can we go with something less generic?

Reply:
Yes, https://dealer-service.preprod.carforyou.ch/swagger-ui/index.html#/Dealer/createPromotionUsingPOST

Reply:
Can you check with them? It should be a low effort to rename a database field and if it makes our code more readable it's definitely worth it.

Reply:
And how would you rename them?

Reply:
maybe

```
{ data: {
  de: {},
  fr: {},
 ...
}
```

but I'd do whatever backend is comfortable with. would not change the database if we follow this naming pattern in other places already

Reply:
@Averethel I checked with BE, is not possible, they implemented it this way e2e